test/google_crawler_web/controllers/upload_controller_test.exs

```diff
@@ -0,0 +1,61 @@
+defmodule GoogleCrawlerWeb.UploadControllerTest do
+  use GoogleCrawlerWeb.ConnCase, async: true
+  alias GoogleCrawler.Repo
+  alias GoogleCrawler.Search.Keyword
+
+  setup :register_and_log_in_user
+
+  describe "GET #new" do
+    test "renders the new page", %{conn: conn} do
+      response =
+        conn
+        |> get(Routes.upload_path(conn, :new))
+        |> html_response(200)
+
+      assert response =~ "Upload a new file!"
+    end
+  end
+
+  describe "POST #create" do
+    test "creates the keywords when file is succesfully processed", %{conn: conn} do
+      upload = %Plug.Upload{path: "test/support/fixtures/csv/valid.csv", filename: "upload.csv"}
+
+      post(conn, Routes.upload_path(conn, :create), %{upload: %{file: upload}})
```

Comment:
should we assert the response status in controller tests 💭

Reply:
Added the test below along with flash message